provision/provision.sh

```diff
@@ -57,19 +57,17 @@ export VVV_PACKAGE_REMOVAL_LIST=()
 . "/srv/provision/core/nginx/provision.sh"
 . "/srv/provision/core/memcached/provision.sh"
 . "/srv/provision/core/php/provision.sh"
-. "/srv/provision/core/composer/provision.sh"
-. "/srv/provision/core/nodejs/provision.sh"
-. "/srv/provision/core/grunt/provision.sh"
 . "/srv/provision/core/mailhog/provision.sh"
-. "/srv/provision/core/wp-cli/provision.sh"
-. "/srv/provision/core/phpcs/provision.sh"
+. "/srv/provision/core/nodejs/provision.sh"
 
 ### SCRIPT
 #set -xv
 
 vvv_hook init
 
+# If you need to disable this check then something is terribly wrong, tell us on github/slack
```

Comment:
maybe is better to add a parameter in `config.yml` so people don't need to hack with the VVV code?

Reply:
We've discussed this elsewhere and the conclusion was "tell us how it failed and we can fix it", if the feature si broken we should fix it, adding an option called `bypass_broken_check: true` is just silly